src/components/Layouts/ReportLayout.less

```diff
@@ -95,11 +94,6 @@ h1 {
             }
           }
         }
-        &.section-show-empty-state {
-          .section-table {
-            height: inherit !important;
-          }
-        }
       }
```

Comment:
why did we removed this ?

Reply:
It is not needed. you can see from the pictures that table without items looks well (the border needs to be until the end of the section)